app/controllers/application_controller/ci_processing.rb

```diff
@@ -1619,6 +1619,42 @@ def scanhosts
     host_button_operation('scan', 'Analysis')
   end
 
+  # Scan all selected or single displayed image(s)
+  def scanimages
```

Comment:
all the (same) functions in there are names the same, so I used the project's convention.